src/dmd/aggregate.h

```diff
@@ -159,6 +159,7 @@ class StructDeclaration : public AggregateDeclaration
 public:
     bool zeroInit;              // !=0 if initialize with 0 fill
     bool hasIdentityAssign;     // true if has identity opAssign
+    bool hasBlitAssign;         // true if opAssign is a blit
     bool hasIdentityEquals;     // true if has identity opEquals
     bool hasNoFields;           // has no fields
```

Comment:
Those booleans looks like they would be a good candidate for an `enum`. In a later PR, obviously.